Lib/test/test_tkinter/test_geometry_managers.py

```diff
@@ -292,8 +292,9 @@ def create2(self):
     def test_place_configure_in(self):
         t, f, f2 = self.create2()
         self.assertEqual(f2.winfo_manager(), '')
-        with self.assertRaisesRegex(TclError, "can't place %s relative to "
-                                    "itself" % re.escape(str(f2))):
+        with self.assertRaisesRegex(TclError, "can't place "
+                                    '"?%s"? relative to itself' %
+                                    re.escape(str(f2))):
```

Comment:
```suggestion
        with self.assertRaisesRegex(
                TclError,
                """can't place "?%s"? relative to itself"""
                 % re.escape(str(f2))):
```
An alternative would be to define `s = re.escape(str(f2)))` above `with` and put `% s` after pattern string.